ui/src/singletons/store.ts

```diff
@@ -312,3 +380,7 @@ export const getNodesView: () => Readonly<EipFlowNode[]> = () =>
   useStore.getState().nodes
 export const getEdgesView: () => Readonly<Edge[]> = () =>
   useStore.getState().edges
+export const getLayoutOrientation: () => Readonly<Layout["orientation"]> = () =>
+  useStore.getState().layout.orientation
+export const getLayoutDensity: () => Readonly<Layout["density"]> = () =>
+  useStore.getState().layout.density
```

Comment:
I think we can decrease the number of methods here by just returning the parent `layout` object. Same for the `useGetLayoutOrientation` and `useGetLayoutDensity` methods.